components/body/index.jade

```diff
@@ -0,0 +1,55 @@
+.main-content-container.main-side-margin
+  .section-one
```

Comment:
Don't forget to namespace by component/app name! It's a tricky habbit to get in to but it would be better if these were `.body-section-one` and `.body-content-container` etc.